utils/pstn-audio-lambdas/amazon-chime-pstn-audio-lambda-node/template.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  amazon-chime-pstn-audio-lambda-node
+
+  Sample SAM Template for amazon-chime-pstn-audio-lambda-node
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 15
+
+Resources:
+  PSTNAudioHandler:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: .
+      Handler: app.lambdaHandler
+      Runtime: nodejs14.x
+      Architectures:
+        - arm64
+
+Outputs:
+  PSTNAudioHandler:
+    Description: 'PSTNAudioHandler Lambda Function ARN'
+    Value: !GetAtt PSTNAudioHandler.Arn
+  PSTNAudioHandlerIamRole:
+    Description: 'Implicit IAM Role created for PSTNAudioHandler function'
+    Value: !GetAtt PSTNAudioHandlerRole.Arn
```

Comment:
Just to confirm. Is `PSTNAudioHandlerRole` automatically generated for the `PSTNAudioHandler` resource?

Reply:
That's correct.  It's just part of the SAM template and generated by SAM.